icu4c/source/common/cmemory.h

```diff
@@ -775,7 +775,7 @@ class MaybeStackVector : protected MemoryPool<T, stackCapacity> {
         return this->fCount;
     }
 
-    T** getAlias() {
+    T** getAlias() const {
```

Comment:
It looks like you need more consts.  It would be better to add a new method like `const T** getConstAlias() const` so that you have all the consts you need.

Reply:
Done.